tests/e2e/e2e_ibc_test.go

```diff
@@ -0,0 +1,382 @@
+package e2e
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/ory/dockertest/v3"
+	"github.com/ory/dockertest/v3/docker"
+)
+
+func (s *IntegrationTestSuite) connectIBCChains() {
+	s.T().Logf("connecting %s and %s chains via IBC", s.chainA.id, s.chainB.id)
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+	defer cancel()
+
+	exec, err := s.dkrPool.Client.CreateExec(docker.CreateExecOptions{
+		Context:      ctx,
+		AttachStdout: true,
+		AttachStderr: true,
+		Container:    s.hermesResource.Container.ID,
+		User:         "root",
+		Cmd: []string{
+			"hermes",
+			"create",
```

Comment:
this "hermes create channel" might induce "mismatch sequence" err due to the competing of this hermes instance with the instance of "hermes start", so it is better to use ""hermes tx chan-open-init" which is already in main branch.

Reply:
i see you also keep "hermes tx chan-open-init" , so the  "hermes create channel"  is not used ?